document-portal/file-transfer.c

```diff
@@ -228,6 +229,7 @@ file_transfer_add_file (FileTransfer *transfer,
 
   file = g_new (ExportedFile, 1);
   file->path = g_strdup (path);
+  file->is_dir = S_ISDIR(parent_st_buf->st_mode) != 0;
```

Comment:
Nit: space before `(`

Comment: I see in other places that `S_ISDIR` is used as a boolean already, maybe you don't need the `!= 0` after all?

Reply:
good catch. it came from code that changed. This is definitely a boolean.